saltproc/tests/test_depcode.py

```diff
@@ -84,11 +95,20 @@ def test_get_nuc_name():
 
 def test_read_depcode_info():
     serpent.read_depcode_info()
-    assert serpent.sim_info['serpent_version'] == 'Serpent 2.1.31'
+    assert serpent.sim_info['depcode_name'] == 'Serpent'
+    assert serpent.sim_info['depcode_version'] == '2.1.31'
     assert serpent.sim_info['title'] == 'Untitled'
+    assert serpent.sim_info['depcode_input_filename'] == \
+        '/home/andrei2/Desktop/git/saltproc/develop/saltproc/data/saltproc_tap'
```

Comment:
Is this right? the path is to andrei's directory?

Reply:
same question for the next two asserts

Reply:
See my above comment